src/codeu/chat/server/View.java

```diff
@@ -45,12 +45,21 @@
 
   private final Model model;
 
+  private static final ServerInfo info = new ServerInfo();
+
   public View(Model model) {
     this.model = model;
   }
-  
+
   // This is implemented to avoid lots of checks with BasicView objects that should be views
   // If the BasicView object is not an instance of a view then it should throw a UnsupportedOperationException.
+
+  @Override
+  public ServerInfo getInfo() { /* Accesses ServerInfo, specifically
+    it returns 'startTime' */
```

Comment:
comment is a bit misleading since serverinfo contains both the version and start time so I would put a comment saying something like "getter method for a server info object"